packages/expo-development-client/ios/EXDevelopmentClientLoadingView.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#if __has_include(<React/RCTDevLoadingViewProtocol.h>)
+
```

Comment:
I'm not sure if we will prebuild the development client package at some point (probably never), but it's more for your information that importing `Foundation` would be necessary to be able to prebuild the library (`NS_ASSUME_NONNULL_BEGIN` is there).